src/utils/__tests__/zulipVersion-test.js

```diff
@@ -67,6 +68,17 @@ describe('ZulipVersion.prototype.raw()', () => {
   });
 });
 
+describe('ZulipVersion.prototype.toJSON()', () => {
+  const rawUnrealistic = randString();
+  const rawRealistic = eg.recentZulipVersion.raw();
+  test('Returns the same string the instance was constructed with', () => {
+    // Perhaps we don't need to check this?
+    expect(new ZulipVersion(rawUnrealistic).toJSON()).toBe(rawUnrealistic);
```

Comment:
It's not the most critical test, but I think it's one worth having. Rather than "unrealistic", one can think of it as testing that the `toJSON` method round-trips a random string, one that lacks any of the structure we're expecting these version strings to have.

IOW, while the other test case checks that `toJSON` doesn't get confused and forget something in the presence of the expected version structure, this one checks that it also doesn't rely on that structure so as to be lost without it.

(In the implementation, both of those properties are accomplished by having it just use the original raw string directly and not even look at the data from having parsed the version string. But one could imagine an implementation that instead put it back together from the parsed pieces, and in a situation not too different from this one that might even be the right choice. Then a bug could well cause either one of these test cases to fail while the other passed.)